content/md/productivity/get-started-with-the-rules-engine.md

```diff
@@ -142,15 +161,51 @@ The expected values are:
 
 To set the *“My very new description for purple T-shirt”* value to your `description` attribute in the `en_US` locale and for the `e-commerce` channel, the action will be as follows:
 
+For instance, the following actions will **disable** the product and set its **family** to `shoes`. It will also **categorize** it in `casual` and `women` (while uncategorizing it from its previous categories), and **add** it to the `summer` group (while removing it from its previous groups).
+
 ```YML
   actions:
-      - type:   set
-          field:  description
-          locale: en_US
-          scope:  ecommerce
-          value:  "My very new description for purple tshirt"
+      - type: set
+        field: enabled
+        value: false
+      - type: set
+        field: family
+        value: shoes
+      - type: set
+        field: categories
+        value:
+           - casual
+           - women
+      - type: set
+        field: groups
+        value:
+          - summer
 ```
 
+#### Set associations
+
+Just like for the add action, you can choose to associate any combination of **products**, **product_models** or **groups** for each association type.
+
```

Comment:
For each association type, you can now decided which associated product, product_model or groups you want to update. The other ones will not be updated.